docs/src/css/custom.css

```
@@ -514,6 +940,159 @@ html[data-page='home'][data-theme='dark'] .dropdown>.navbar__link:after {
+  display: none;
+}
+
+/* Increase space between main content and right sidebar TOC */
+.theme-doc-toc-desktop {
+  padding-left: 0.75rem;
+  margin-top: 2.7rem;
+  font-size: 1.5em;
+  line-height: 1.5;
+}
+
+.theme-doc-toc-desktop .table-of-contents__link {
```

Stylelint (17.11.0)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)